package/base-files/files/etc/init.d/led

```diff
@@ -49,6 +55,14 @@ load_led() {
 	[ -e /sys/class/leds/${sysfs}/brightness ] && {
 		echo "setting up led ${name}"
 
+		[ "${uci_brightness}" -lt 0 ] || [ "${uci_brightness}" -gt "${uci_max_brightness}" ] &&
+			uci_brightness="${uci_max_brightness}"
+
+		# Convert UCI brightness by doing a cross product (uci_brightness / uci_max_brightness) = (x / sysfs_max_brightness).
+		# The cross product is rounded up to avoid turning off the led even if there is a valid positive brightness configured.
+		read sysfs_max_brightness < "/sys/class/leds/${sysfs}/max_brightness"
+		sysfs_brightness=$(( ( $uci_brightness * $sysfs_max_brightness + $uci_max_brightness - 1 ) / $uci_max_brightness ))
```

Comment:
just a tiny nit-pick, but why the extra spaces around parenthesis? If this is luci coding style it's fine, but I thought it wasn't :) (e.g. they only belong around the operators. (I'd add even more, to visually make it clear the multiplication is expected to go first, I don't trust 'math rules' :p)

Also, I think when you have a 'math subshell', you don't need the dollar signs at all.

Reply:
extra space increases readability and countability of how many level deep in you are :)

Reply:
'sometimes' :p it really depends. In this case, to me _personally_ it introduces noise :p but that's where coding standards come in to place :)